### PR TITLE
Feat: Events Component Implemented

### LINKED ISSUE #90: Coding: Develop the following Events Section
## Description
As a developer,
I need the following section to be developed,
so that I can add it as an Events component [here](https://github.com/anitab-org/anitab-org.github.io/tree/develop/src/Components).

## Mocks
Develop the section in the red box only
event1

Font Size for heading EVENTS: Header Medium(32pt) weight- Medium
Font Size for the description below: body 18pt weight- Light
Font Style- Avenir
Font Color (heading): #103B81
Font Color (Body): #103B81

[Design Link for Development](https://xd.adobe.com/view/237a598d-ce4a-4fee-5642-3048a38e6a8b-554b/)

### Update [Required]
- [ ] Should follow all the margin guidelines mentioned above and should be responsive

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
6-8 hours

### PR DESCRIPTION
**Pull request by prabhat04011998**
### Description

I have implemented the OurEvents Component ,taking the data from ourevents.json. which will reflected on the events page of the web-app.

Fixes  [#90 ]
closes #90 

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**

- Bug fix (non-breaking change which fixes an issue #90)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
I have tested the implementation and responsiveness.
here is the deployed link.
https://prabhat04011998.github.io/anitab-org.github.io/

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have attached link of deployed site.
- [x] No dependencies added.

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by keshakaneria** [comment]
@prabhat04011998 will check and review soon :)

**Comment by prabhat04011998** [comment]
@keshakaneria  thanks for reviewing.

**Comment by devkapilbansal** [comment]
@Akanksha1212 you should not give two labels to a single PR. As far as I know, `Needs Testing` label is given after review is completed

**Comment by Akanksha1212** [comment]
Thanks for pointing it out @devkapilbansal. Anyways testing part is done as the contributor has already provided the deployment link.

**Comment by prabhat04011998** [comment]
Dear Reviewers,
Guys it's been a long time , please review my PR,

**Comment by devkapilbansal** [comment]
Hi @prabhat04011998 you should wait for your PR to be reviewed. In that time you can look in more issues, tests the application or review others pull request. Please be patient

**Comment by nandini45** [comment]
I think its just a bug fix @Akanksha1212  probably check it once and merge it. It won't be a issue.

**Comment by prabhat04011998** [comment]
Anyone Reviewing the PR ?

**Comment by prabhat04011998** [comment]
@keshakaneria @Akanksha1212  
guys can you review this pr , if you forgot,
it'll be the great help if you guys can get some time over it , it's small changes it'll not take your much time.
Thanks :)

**Review by keshakaneria — APPROVED** [review]
Seems good! Just squash the commits before it's ready to merge :)

**Review by nandini45 — APPROVED** [review]
@prabhat04011998 sorry for the late response 
Thanks for the contribution

**Comment by keshakaneria** [comment]
> @keshakaneria thank for your review.. Anna is a bit busy nowadays.
> As you suggested all the changes are correct i will approve and merge this PR

Awesome, go for it :)

**Comment by prabhat04011998** [comment]
@keshakaneria  @nandini45  i think not merged yet .Just a healthy reminder.

**Comment by tichnas** [comment]
I think this PR contains commits meant for another issue (#166)

**Review by tichnas — CHANGES_REQUESTED** [review]
I think this PR contains commits meant for another issue (#166)

**Comment by prabhat04011998** [comment]
@tichnas  no i think it was just a single line.Any issues in merging?

**Comment by tichnas** [comment]
Yes, it's just a single line. There is as such no issue in merging, but it's a better practice to create one PR for only one issue.

**Review by prabhat04011998 — COMMENTED** [review]
It's a single line, i'll take care from now onwards @tichnas

**Comment by tichnas** [comment]
@prabhat04011998 I completely understand it's quite a pain to do these small changes when you're just starting, but it's worth it when it comes to a Repo as big as this where best practices matter a lot.

Try to do it (maybe some googling also), let me know what problems you face, and I'll be there to help :smiley:

P. S. I'm just a contributor and can't merge anyway. But I suggest you to make your PR so perfect that when admins see it, they merge it directly without requesting any changes (this will save you a lot of time and as I said these practices are best for long term maintenance, even if it's your personal project).

**Comment by prabhat04011998** [comment]
@tichnas squashed commits done.

**Comment by keshakaneria** [comment]
> @tichnas squashed commits done.

The commit count still shows 6. Once you've squashed them, you will be able to see their the count as only 1. You can try again squashing them, it seems squashing is unsuccessful :)

**Comment by prabhat04011998** [comment]
@keshakaneria  @nandini45 I have removed the code for another issue , 
actually new branch was not created and things are not working while squashing , please admin can squash and merge this time , and next time i'll take care of this.

File: src/Components/Header/index.js [hunk 1]
```diff
@@ -64,7 +64,8 @@ function MenuItem(index, selected, setSelected, title) {
 const styles = StyleSheet.create({
   logoContainer: {
     flex: 1,
-    marginRight:80,
```
Comment (annabauza) [inline src/Components/Header/index.js:67]: This is generic component and will change all the headers.

**Review by annabauza — CHANGES_REQUESTED** [review]
I think this issue will create regression.

File: src/Components/Header/index.js (hunk 1 above)
Comment (tichnas) [reply src/Components/Header/index.js:67]: @annabauza The issue which was meant to be solved in this PR is already solved in #176, so I think we should close this PR now?

Comment (prabhat04011998) [reply src/Components/Header/index.js:67]: @annabauza  the issue was in the whole header so we have to change in the generic component for that and it's fixed.

**Review by annabauza — APPROVED** [review]

**Pull request merged**